back/db/migrate/20210601061247_create_category_assignments.insights.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+# This migration comes from insights (originally 20210526132209)
+
+class CreateCategoryAssignments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :insights_category_assignments, id: :uuid do |t|
+      t.references :category, type: :uuid, null: false, index: true, foreign_key: { to_table: :insights_categories }
+      t.references :input, type: :uuid, null: false, index: true, polymorphic: true
+      t.boolean :approved, null: false, index: true, default: true
+
```

Comment:
Not sure to understand how this would help. Do you have minimal example in mind?

I already started to implement the search in this [PR](https://github.com/CitizenLabDotCo/citizenlab/pull/302).

Reply:
When you're looking at an input in the side modal, there's a field you can type in to add a category to an input, if no category was found with what you typed, this category can be created and assigned instead. Without the nested attributes, front-end will have to make two requests : create the category, await the result and then create the assignment.

Reply:
Ah I see! Gonna log this one to Jira not to block this PR.

Reply:
https://citizenlab.atlassian.net/browse/IN-405

Reply:
Sounds good, thanks ! :)